presto-product-tests/conf/docker/files/apply-site-xml-override.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -euo pipefail
```

Comment:
we should bake this utility to base docker image so we can refactor our images.

Reply:
Good idea. Please create an issue.

Reply:
https://github.com/prestosql/docker-images/issues/40